src/pages/Home.vue

```diff
@@ -1,6 +1,126 @@
 <template>
   <main>
-    <h1>LevelDB Inspector</h1>
-    <img src="@images/logo.png" alt="App Logo" />
+    <header>
+      <div class="logo"></div>
+    </header>
+    <form>
+      <p>Select path to LevelDB</p>
+      <label class="file db-path">
+        <input type="file" id="file" aria-label="File browser example" />
+        <span class="file-custom"></span>
+      </label>
+      <input type="submit" value="Inspect" />
+    </form>
+    <footer>Copyright 2020 • UncaughtException</footer>
   </main>
 </template>
+
+<style scoped>
+main {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  text-align: center;
+}
+
+.logo {
+  width: 150px;
+  height: 150px;
+  background-color: #f7f7f7;
+  border-radius: 100%;
+  margin: 100px auto;
+}
+
+form {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+
+/*
+ * http://wtfforms.com/
+ */
+
+.file {
+  position: relative;
+  display: inline-block;
+  cursor: pointer;
+  height: 2.5rem;
```

Comment:
I'm no expert in css but I think if we use `rem` we should define the font size of the `html` element instead of relying on the browser default

Reply:
I couldn't find definitive advice on setting font size of html element vs default. In any case, I switched all sizing to ems since I think in this case it's better that those measurements scale with current element font size, instead of root.

Reply:
@jjlumagbas I'm sure I've read all of these from somewhere but generally browsers use a default font size of `16px`. Using `rem` against this default value becomes tricky sometimes because you need to be calculating e.g. `1.5rem = 1.5 * 16px` etc. What I personally like doing is to do:

```css
html {
    font-size: 62.5%;
}
```

This will set the root element's font size to `10px`, which is a lot easier to compute values with that `16px`, e.g. `1.5rem = 15px`. There are use cases for `em` as well, but I don't use it that much. But regardless of which one we use, `rem` or `em`, that would be better than the absolute units (`px`, etc.) because `rem`/`em` respects the user's font size preferences on their browsers. 👍

Reply:
Should we standardize our approach @arnellebalane? I think this approach seems reasonable:

> 1. Size in em if the property scales according to its font-size
> 2. Size everything else in rem.

https://zellwk.com/blog/rem-vs-em/

So ems by default and then rems to address any relative alignment issues?

Reply:
@jjlumagbas Yes we should 😅 I'm fine with this: 

> So ems by default and then rems to address any relative alignment issues?